xml/release-notes.xml

```diff
@@ -513,31 +513,23 @@
 
  </sect1>
 
- <!-- <sect1 xml:id="sec.driver">
+ <sect1 xml:id="sec.driver">
   <title>Drivers and Hardware</title>
   <para/> -->
 
-  <!-- <sect2 xml:id="sec.driver.nvidia-hybrid">
-   <!-/- boo#1094834 -/->
-   <title>Hang on Machines with Nvidia GPUs and Hybrid Graphics</title>
+  <sect2 xml:id="sec.driver.sig">
+   <!-/- bsc#173158 -/->
+   <title>CONFIG_MODULE_SIG=y</title>
    <para>
-    With the kernel shipped in &thisflavor; 15.0 GM, the Nouveau driver for
-    Nvidia graphics card may hang at reboot, shutdown, or during runtime power
-    management actions. This bug occurs primarily on system with hybrid
-    graphics, such as laptops that include integrated Intel graphics and a
-    discrete Nvidia graphics card.
+     This is an important change for all users of 3rd party drivers using secureboot.
```

Comment:
How about adding an example after "3rd party drivers"? E.g. "(for example NVidia or AMD graphics drivers from the respective manufacturer's repositories)"